mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -16,6 +16,30 @@ type mockRDSClient struct {
 	dbSnapshots          []*rds.DBSnapshot
 }
 
+// AddSnapshots add a list of snapshots to the mock, both in the full
+// list and in the per instance map.
+func (m *mockRDSClient) AddSnapshots(
```

Comment:
Could we maybe give `m` a bit more descriptive name?

Also, what does the `*` mean? Is it a pointer to a mock RDC client?